fiftyone/utils/coco.py

```diff
@@ -1270,7 +1270,10 @@ def load_coco_detection_annotations(json_path, extra_attrs=True):
 
 def _parse_coco_detection_annotations(d, extra_attrs=True):
     # Load info
-    info = d.get("info", {})
+     info = {}
```

Comment:
I'd recommend writing it like this:

```py
info = d.get("info", None)
if info is None:
    info = {}
```

It's a bit easier to read 😄

Reply:
Thanks Just made the change